ansible/roles/test/tasks/pfc_wd/functional_test/functional_test_restore.yml

```diff
@@ -0,0 +1,142 @@
+#------------------------------------
+# Test the PFC restore action
+# 1. Verify restore is detected via syslog entry analysis on the dut
+# 2. Verity the forward action on both ingress and egress via ptf dataplane traffic test
+#------------------------------------
+
+- name: Set fact for item values
```

Comment:
This is a simplified version of the PFC watchdog test. Here we only verify:
For a storm event:
1) control plane detection message in syslog; 2) data plane traffic drop in both directions

For a restore event:
1) control plane restore message in syslog; 2) data plane traffic forward in both directions

Reply:
Writing the ansible test makes me crazy. The version is too old. I have run into situation in writing the codes that even if the code is syntax right, the ansible test just runs with error and stops. My policy is minimum touch on the existing codes to avoid asking for trouble.